zone/raids.cpp

```diff
@@ -2203,3 +2257,734 @@ void Raid::SetNewRaidLeader(uint32 i)
 		}
 	}
 }
+
+void Raid::SaveRaidNote(const char* who, const char* note) {
```

Comment:
These should be `std::string who, std::string note`

Then you can do things like `if (who.empty() || note.empty())` below

Also removes the need to do the `c_str()` castings